docs/learn/expert-guides/key-manager/get-controller-permissions.md

```diff
@@ -5,185 +5,169 @@ sidebar_position: 1
+{
+  key: '0x4b80742de2bf82acb3630000a1061408e55c971fd129ef5561dfb953d598dad7',
+  name: 'AddressPermissions:Permissions:a1061408e55c971fD129eF5561dFB953d598dAD7',
+  value: '0x0000000000000000000000000000000000000000000000000000000000000008'
+}
+```
 
-// setup
-const myUniversalProfileAddress = '0xC26508178c4D7d3Ad43Dcb9F9bb1fab9ceeD58B5';
-const RPC_ENDPOINT = 'https://rpc.testnet.lukso.network';
+</Details>
 
-const web3 = new Web3(RPC_ENDPOINT);
+## Decode Controller Permissions
 
-// step 1 - setup erc725.js
-const erc725 = new ERC725(
-  LSP6Schema,
-  myUniversalProfileAddress,
-  web3.currentProvider,
-);
+Permission are encoded as a [`BitArray`](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md#BitArray). After queriying their values, you can decode them by calling the and [`decodePermissions(...)`](../../../tools/erc725js/classes/ERC725#decodepermissions) function of the 🛠️[`erc725.js`](../../../tools/erc725js/getting-started.md) library:
 
+```js
 async function getPermissionedAddresses() {
-  // step 2 - get the list of addresses that have permissions on the Universal Profile
-  const result = await erc725.getData('AddressPermissions[]');
 
-  for (let ii = 0; ii < result.value.length; ii++) {
-    const address = result.value[ii];
+  // ...
 
-    // step 3.1 - get the permissions of each address
-    const addressPermission = await erc725.getData({
-      keyName: 'AddressPermissions:Permissions:<address>',
-      dynamicKeyParts: address,
-    });
+  if (Array.isArray(result.value)) {
 
-    // step 3.2 - decode the permission of each address
-    const decodedPermission = erc725.decodePermissions(addressPermission.value);
+    // ...
 
-    // we use JSON.stringify to display the permission in a readable format
+    // Decode the permission of each address
+    const decodedPermission = erc725.decodePermissions(addressPermission.value as string);
+
+    // Display the permission in a readable format
     console.log(
-      `decoded permission for ${address} = ` +
-        JSON.stringify(decodedPermission, null, 2),
+      `Decoded permission for ${address} = ` + JSON.stringify(decodedPermission, null, 2),
     );
   }
 }
-
 getPermissionedAddresses();
 ```
+
+<Details>
+    <summary>Show <code>decodedPermission</code> value</summary>
+
+```js
+Decoded permission for 0x5F606b5b237623463a90F63230F9b929321dbCBa = {
+  "CHANGEOWNER": true,
+  "ADDCONTROLLER": true,
+  "EDITPERMISSIONS": true,
+  "ADDEXTENSIONS": true,
+  "CHANGEEXTENSIONS": true,
+  "ADDUNIVERSALRECEIVERDELEGATE": true,
+  "CHANGEUNIVERSALRECEIVERDELEGATE": true,
+  "REENTRANCY": false,
+  "SUPER_TRANSFERVALUE": true,
+  "TRANSFERVALUE": true,
+  "SUPER_CALL": true,
+  "CALL": true,
+  "SUPER_STATICCALL": true,
+  "STATICCALL": true,
+  "SUPER_DELEGATECALL": false,
+  "DELEGATECALL": false,
+  "DEPLOY": true,
+  "SUPER_SETDATA": true,
+  "SETDATA": true,
+  "ENCRYPT": true,
+  "DECRYPT": true,
+  "SIGN": true,
+}
+```
```

Comment:
is this js code? 🤔 

<img width="107" alt="image" src="https://github.com/lukso-network/docs/assets/477945/f538abbd-8521-41a4-9ede-dd30a7b20a3b">

<img width="697" alt="image" src="https://github.com/lukso-network/docs/assets/477945/2836d677-bea9-4a5b-8ac2-1873c4ae098d">